### PR TITLE
[forge] turn on run-forge-single-vfn-perf

### PR DESCRIPTION
**Pull request by bchocho**
### Description

Introduced in #3363 . Run every 12 hours

### Test Plan

Manually triggered to see the test succeeds https://github.com/aptos-labs/aptos-core/actions/runs/2930950790

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/3493)

### CODE REVIEW

File: .github/workflows/continuous-e2e-single-vfn-test.yaml [hunk 1]
```diff
@@ -6,10 +6,12 @@ permissions:
 
 on:
   workflow_dispatch:
+    schedule:
+      - cron: "*/12 * * * *"
 
 jobs:
   # Test transaction processing throughput at a single VFN
-  run-forge-chaos:
+  run-forge-single-vfn-perf:
```
Comment (bchocho) [inline .github/workflows/continuous-e2e-single-vfn-test.yaml:14]: Fixed the name, which was wrong

**Review by sitalkedia — APPROVED** [review]

File: .github/workflows/continuous-e2e-single-vfn-test.yaml [hunk 2]
```diff
@@ -19,5 +21,4 @@ jobs:
       FORGE_RUNNER_DURATION_SECS: 480
       FORGE_RUNNER_TPS_THRESHOLD: 5000
       FORGE_TEST_SUITE: single_vfn_perf
-      POST_TO_SLACK: true
```
Comment (rustielin) [inline .github/workflows/continuous-e2e-single-vfn-test.yaml:22]: why not post to slack :^)

Comment (bchocho) [reply .github/workflows/continuous-e2e-single-vfn-test.yaml]: Good catch. I think I tried to see if the previous permissions issue was due to slack :P

**Comment by bchocho** [comment]
@rustielin can you be the second accepter?

**Review by perryjrandall — APPROVED** [review]

**Comment by bchocho** [comment]
Thanks for catching the cron mistake @rustielin !

**Pull request merged**